packages/@aws-cdk/aws-codepipeline-actions/test/codestar-connections/codestar-connections-source-action.test.ts

```diff
@@ -148,27 +148,155 @@ describe('CodeStar Connections source Action', () => {
       ],
     });
 
+  });
+
+  test('exposes variables', () => {
+    const stack = new Stack();
+
+    createBitBucketAndCodeBuildPipeline(stack, {
+      triggerOnPush: false,
```

Comment:
Any reason we are passing this property in this test?

Reply:
I just copied and pasted that from another block of code if I remember correctly. I will have it changed with next commit and refactor for utility method requested in another comment.